js_modules/dagster-ui/packages/ui-core/src/runs/RunTimelineRowIcon.tsx

```diff
@@ -17,6 +18,8 @@ export const RunTimelineRowIcon = ({type}: {type: RowObjectType}) => {
       return <Icon name="sensors" />;
     case 'legacy-amp':
       return <Icon name="sensors" />;
+    case 'airflow':
+      return <TagIcon label="airflow" />;
```

Comment:
can we not use `Icon` here now?

Reply:
can't use Icon because it renders in monochrome. I actually need to get rid of all the icon changes.